src/sentry/api/endpoints/organization_projects.py

```diff
@@ -82,6 +82,18 @@ def get(self, request, organization):
                 organization=organization,
             ).prefetch_related('teams')
 
+        if request.user.is_authenticated():
+            queryset = queryset.extra(
+                select={
+                    'is_bookmarked': """exists (
+                        select *
+                        from sentry_projectbookmark spb
+                        where spb.project_id = sentry_project.id and spb.user_id = %s
+                    )""" % request.user.id,
```

Comment:
Why do you string interpolate the user.id and provide it in `select_params` as well?

Reply:
Oops, forgot to remove this string formatting here.